Pipfile

```diff
@@ -7,27 +7,19 @@ name = "pypi"
 lint = "flake8"
 
 [packages]
-raven = "==6.10.0"
+uvicorn = "==0.11.8"
```

Comment:
Uma boa pratica é utilizar ~= em vez de ==
Seguindo as regras da PEP-440

mais informações:
https://stackoverflow.com/questions/39590187/in-requirements-txt-what-does-tilde-equals-mean
https://www.python.org/dev/peps/pep-0440/